pkg/plan_modifier/data_group_custom_diff.go

```diff
@@ -209,6 +209,8 @@ func (m customDataGroupDiffModifier) PlanModifySet(ctx context.Context, req plan
 			}
 
 			// pg config
+			// TODO: We should add the default pg_config values to the plan, so that we show the correct drift.
```

Comment:
I'm going to create a ticket for this one.

Reply:
I created https://github.com/EnterpriseDB/terraform-provider-biganimal/issues/315 and assigned to you, @wai-wong-edb.